logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -17,6 +17,23 @@
 import math
 import json
 
+try:
+    import django
+except ImportError:
+    _USE_DJANGO = False
+else:
+    _USE_DJANGO = True
+
+try:
+    import flask
+except ImportError:
+    _USE_FLASK = False
+    flask = None
+else:
+    _USE_FLASK = True
+
```

Comment:
Overall this strategy seems risky since you might have Flask installed in a Django app's environment which would cause a confusing error. I would consider if both `USE_FLASK` and `USE_DJANGO` are true, raise some sort of error. I would also consider adding an environment variable that explicitly chooses, and which could be used to fix the above error.